lib/shared/robot/index.js

```diff
@@ -119,19 +116,9 @@ exports.parseMessage = (string) => {
  *
  * console.log(error.data.message);
  * > 'foo'
- *
- * error.data.stacktrace === error.stack;
- * > true
  */
 exports.buildErrorMessage = (error) => {
```

Comment:
I suggest keeping it separate for testability purposes. `.printError` does a `console.log`, which will be hard to test.